users/tests.py

```diff
@@ -1,3 +1,54 @@
-# from django.test import TestCase
+from datetime import date
+import pytest
+from django.contrib.auth import get_user_model
+from users.models import Profile
 
-# Create your tests here.
+USER_NAME = 'testuser'
+USER_PASSWORD = 'testpass'
+PHONE_NUMBER = '1234567890'
+PHONE_NUMBER_NEW = '0987654321'
+
+
+@pytest.fixture
+def user1():
+    user = get_user_model().objects.create(username=USER_NAME, password=USER_PASSWORD)
+    return user
```

Comment:
maybe it will be better to move the save to here instead of calling it a few times later 
```suggestion
    user.save()
    return user
```